content/02.main-text.md

```diff
@@ -86,7 +83,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
 These discussions must be archived to ensure that important decisions about the manuscript are preserved and authors receive credit for intellectual contributions that are not directly reflected in the manuscript's text.
 GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
 
-`TODO: cgreene paragraph on diversity, open science culture, possibly author disagreements and conduct`
+It is important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
+Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1080/08874417.2009.11645338].
+Conferences and open source projects have used codes of conduct to establish these norms [@url:http://geekfeminism.wikia.com/wiki/Code_of_conduct_evaluations?oldid=35801; @url:https://www.contributor-covenant.org/].
+In our open review paper, we established contributor guidelines that discussed norms in the areas of of text contribution, peer review, and authorship [@url:https://github.com/greenelab/deep-review/blob/2bb8c3960bc4616f32eaceb3fe06002c51fb2784/CONTRIBUTING.md], which we identified in advance as potential areas of disagreement.
+Our contributor guidelines required verifiable participation: either directly attributable changes to the text or participation in the discussion on GitHub.
+We maintained our guidelines, even when a case arose where two authors had written text together but only one had directly attributable changes and participation.
+Our contributor guidelines did not discuss broader community norms that may have improved inclusiveness.
```

Comment:
"Our contributor guidelines" is repeated.  Maybe use "these guidelines" instead in this line or the previous instance?

Reply:
👍